src/modules/site-v2/templates/_includes/isotype-database.html

```diff
@@ -1,749 +1,80 @@
-              <td>ECA1069</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.89411879</td>
-              <td>-56.15794301</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">45</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1070</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8934668</td>
-              <td>-56.15637202</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">46</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1071</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8934668</td>
-              <td>-56.15637202</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">47</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1072</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8434</td>
-              <td>-56.1199</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">48</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1073</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8434</td>
-              <td>-56.1199</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <!-- /Sample Data -->
-          </tbody>
-        </table>
-      </div>
-      <!-- /Table Placeholder -->
\ No newline at end of file
+{% block custom_head %}
+<style>
+  #isotype-db_filter {
+    margin: 1rem 0 2rem 0
+  }
+  #isotype-db_length {
+    text-align: left;
+  }
+  .badge {
+    color: black
+  }
+  thead {
+    background-color: #e1eae4;
+  }
+
+</style>
+
+{% endblock %}
+
+<div class="table-responsive my-5 isotypeDatabase">
```

Comment:
I'd recommend moving the `table-responsive` div to the home.html template since that just feels a little bit safer and less likely to break the design in the future.

I also think you can remove the `isotypeDatabase` class, because all that's doing is limiting the height. I think there's enough room in the design to show the full height.